examples/rust/opensk/src/clock.rs

```diff
@@ -13,18 +13,43 @@
 // limitations under the License.use opensk_lib::api::clock::Clock;
 
 use opensk_lib::api::clock::Clock;
+use wasefire::clock::Mode::Oneshot;
+use wasefire::clock::{Handler, Timer};
 
 #[derive(Default)]
-pub struct WasefireClock {}
+struct ClockHandler;
+
+impl Handler for ClockHandler {
+    fn event(&self) {
+        // TODO: What should we do here?
+        wasefire::debug!("timer elapsed");
+    }
+}
+
+pub struct WasefireTimer {
+    timer: Timer<ClockHandler>,
```

Comment:
We need a field to know whether the timer as triggered.

```suggestion
    timer: Timer<ClockHandler>,
    elapsed: Rc<Cell<bool>>,
```